products/samsungmobile.md

```diff
@@ -30,27 +135,112 @@ releases:
     support: true
     eol: false
     releaseDate: 2022-04-01
+    
+-   releaseCycle: "Galaxy F23"
+    support: true
+    eol: false # https://doc.samsungmobile.com/SM-E236B/INS/doc.html
+    releaseDate: 2022-03-16
+
+-   releaseCycle: "Galaxy M33"
+    support: true
+    eol: false # https://doc.samsungmobile.com/SM-M336BU/INS/doc.html
+    releaseDate: 2022-04-08
+
+-   releaseCycle: "Galaxy M23"
+    support: true
+    eol: false # https://doc.samsungmobile.com/SM-M236B/XXV/doc.html
+    releaseDate: 2022-04-08
+
+-   releaseCycle: "Galaxy A23"
+    support: true
+    eol: false # https://doc.samsungmobile.com/SM-A235M/PET/doc.html
+    releaseDate: 2022-03-25
+
+-   releaseCycle: "Galaxy A13"
+    support: 2022-10-01 # Approximate from here: https://9to5google.com/2022/10/04/samsung-android-12-update-rollout/
+    eol: false # https://doc.samsungmobile.com/SM-A135M/PET/doc.html
+    releaseDate: 2022-03-23
+
+-   releaseCycle: "Galaxy S22 Ultra 5G"
+    support: true
+    eol: false # https://doc.samsungmobile.com/SM-S908E/VAU/doc.html
+    releaseDate: 2022-02-25
+
+-   releaseCycle: "Galaxy S22+ 5G"
+    support: true
+    eol: false # https://doc.samsungmobile.com/SM-S906E/XXV/doc.html
+    releaseDate: 2022-02-25
+
+-   releaseCycle: "Galaxy S22 5G"
+    support: true
+    eol: false # https://doc.samsungmobile.com/SM-S901E/XXV/doc.html
+    releaseDate: 2022-02-25
 
+-   releaseCycle: "Galaxy Tab S8 Ultra"
+    support: 2022-10-01 # Approximate from here: https://9to5google.com/2022/10/04/samsung-android-12-update-rollout/
```

Comment:
Only 6 months of support !?

Reply:
Looking into this, I think in a later PR we really need to redefine the full support column, but for now I think if a device has received android 12 we mark it as fully supported, and then potentially revisit this with android 13 and mark those that didn't get android 13 as not fully supported anymore. 

This approach would mark the two Tab S8 as fully supported for now, do you agree with this?

Reply:
Yes.